zxcvbn-cs/DefaultMatcherFactory.cs

```diff
@@ -19,18 +20,26 @@ class DefaultMatcherFactory : IMatcherFactory
     {
         List<IMatcher> matchers;
 
+        private IEnumerable<string> DictionaryResources()
+        {
+            var executingAssembly = Assembly.GetExecutingAssembly();
+            string folderName = string.Format("{0}.Resources.Folder", executingAssembly.GetName().Name);
```

Comment:
Document what these changes do in the readme.

Reply:
Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/1/commits/c37b3a0cf43fa2939e8a8402940eaf8dd927b1b2